Kitodo/src/main/java/de/sub/goobi/forms/SessionForm.java

```diff
@@ -37,6 +39,7 @@
     private SimpleDateFormat formatter = new SimpleDateFormat("HH:mm:ss");
     private String aktuelleZeit = this.formatter.format(new Date());
     private String bitteAusloggen = "";
+    private static final Logger logger = LogManager.getLogger(ModuleServerForm.class);
```

Comment:
This class is called `SessionForm` but you are creating a logger for class `ModuleServerForm`